pkg/operator/controller/status/controller_test.go

```diff
@@ -533,32 +535,76 @@ func Test_computeOperatorStatusVersions(t *testing.T) {
 	}
 }
 
+// TODO: Consider separating the test into two: one for ingresscontrollers, another for gatewayapi crds (?)
```

Comment:
It looks fine, `computeOperatorUpgradeableCondition` now takes the CRDs into consideration too.